src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1740,10 +1740,7 @@ internal static class Requires
     {
         internal static void NotNull(object value, string paramName)
         {
-            if (value == null)
-            {
-                throw new ArgumentNullException(paramName);
-            }
+            ArgumentNullException.ThrowIfNull(value, paramName);
```

Comment:
In follow PR we could replace all `NotNull` with the `ArgumentNullException.ThrowIfNull`.